rules/sinks/third_parties/sdk/google/javascript.yaml

```diff
@@ -50,7 +50,7 @@ sinks:
     domains:
       - "admob.google.com"
     patterns:
-      - "((window|w|this)[.])?(gtag|dataLayer.push)[(].*"
+      - "(?i)(.*[.])?(gtag|dataLayer.push)[(].*"
```

Comment:
This will send google tag calls to admob.

Reply:
yes will change.